deprecations/README.md

```diff
@@ -39,6 +39,12 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| `TaxIdentifier` in [Close bill](../operations/bills.md#close-bill) | Replaced by `AccountTaxIdentifier` | 16 Nov 2023 | 16 Nov 2024 |
+| `TaxIdentifier` in [Close bill](../operations/bills.md#close-bill) | Replaced by [AssociatedAccountData](../operations/bills.md#bill-associated-account-data) | 16 Nov 2023 | 16 Nov 2024 |
+| `Address` in [Close bill](../operations/bills.md#close-bill) | Replaced by `AccountAddress` | 16 Nov 2023 | 16 Nov 2024 |
```

Comment:
Why is this deprecated? `Address` is replaced by a new property which is identical except it has the word "Account" added to the name. Am I missing something?

Reply:
So the user doens't use Address anymore but AccountAddress. Address has been annotated as Deprecated. What do you suggest?

Reply:
The fact is that `Address` was already deprecated for some time with the note:

> Use AccountAddress or CompanyAddress instead

Now we're deprecating also `CompanyAddress` in favor of `AccountAddress` which has been around since like February.

Reply:
OK, there are some internal complications here. As discussed, I will approve the change, but I think we should consider in future the impact on the API 'user contract' and avoid unnecessary deprecations.